client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.ts

```diff
@@ -255,6 +255,11 @@ export class MotionMetaDataComponent extends BaseMotionDetailChildComponent {
         return allStates.filter(state => state.recommendation_label).sort((a, b) => a.weight - b.weight);
     }
 
+    public getReferencingMotions(): ViewMotion[] {
+        const allReferences = this.motion.referenced_in_motion_recommendation_extensions || [];
+        return allReferences.sort((a, b) => a.number.localeCompare(b.number));
```

Comment:
Don't put sort functions in a getter. Especially not if it is called in the template.

You could instead:
1. Add the property `referenced_in_motion_recommendation_extensions_as_observable: Observable<OptionData[]>` to the `ViewMotion` class (preferably edit the `HasReferencedMotionsInExtension` interface), this observable will automatically be updated by the base repository when `referenced_in_motion_state_extensions` is.
2. In some central place of this class: Subscribe to `this.motion.referenced_in_motion_recommendation_extensions_as_observable` and sort and write every update into a designated variable
3. Return that variable in this getter

Also this getter might as well be moved above the constructor and declared as `public get referencingMotions(): ViewMotion[] {...`